csrc/scheduler/utils.cpp

```diff
@@ -1088,10 +1088,7 @@ IterDomain* projectIdToRoot(
         }
       }
     } else if (expr->isA<Resize>()) {
-      auto resize = expr->as<Resize>();
-      if (resize->out() == projected_id) {
-        projected_id = resize->in();
-      }
+      projected_id = nullptr;
```

Comment:
Shouldn't this only mater when `resize->out() == projected_id`?

Reply:
Oops, you are right. Fixed!